lib/Twig/Cache/Filesystem.php

```diff
@@ -44,17 +44,9 @@ public function generateKey($name, $className)
     /**
      * {@inheritdoc}
      */
-    public function has($key)
-    {
-        return is_file($key);
-    }
-
-    /**
-     * {@inheritdoc}
-     */
     public function load($key)
     {
-        require_once $key;
+        @include_once $key;
```

Comment:
Checking the existence of a file followed by attempting to load it is prone to a race condition.